examples/ateccx08/lock/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"machine"
+
+	"encoding/hex"
+	"time"
+
+	"tinygo.org/x/drivers/ateccx08"
+)
+
+func main() {
+	time.Sleep(5 * time.Second)
+	println("Looking for ATECCx08...")
+
+	machine.I2C0.Configure(machine.I2CConfig{})
+
+	atecc := ateccx08.New(machine.I2C0)
+	atecc.Configure()
+
+	if !atecc.Connected() {
+		for {
+			println("could not connect to ATECCx08")
+			time.Sleep(time.Second)
+		}
+	}
+
+	version, _ := atecc.Version()
+
+	println(version.String(), "started")
+
+	if !atecc.IsLocked() {
+		for i := 10; i > 0; i-- {
+			println(version.String(), "is not locked. Locking in", i, "seconds...")
+			time.Sleep(time.Second)
+		}
+
+		// locks the Configuration zone... PERMANENTLY!
```

Comment:
Perhaps it's a good idea to put this a bit more prominently at the top of the file? Examples don't usually make permanent changes to hardware.

Reply:
That's true. We could probably even have a boolean constant set to `false` (dryRun?) by default and require it to be changed for irreversible modifications to be actually applied.